src/Explorer.js

```diff
@@ -851,9 +1192,34 @@ class AbstractArgView extends React.PureComponent<AbstractArgViewProps, {}> {
           )}
           <span
             style={{color: styleConfig.colors.attribute}}
-            title={arg.description}>
+            title={arg.description}
+            onMouseEnter={() => {
+              // Make implementation a bit easier and only show 'variablize' action if arg is already added
+              if (argValue !== null && typeof argValue !== 'undefined') {
+                this.setState({displayArgActions: true});
+              }
+            }}
+            onMouseLeave={() => this.setState({displayArgActions: false})}>
             {arg.name}
             {isRequiredArgument(arg) ? '*' : ''}:
+            {!this.state.displayArgActions ? null : (
```

Comment:
Can you move this to the outside of the variable value so that it doesn't shift the param over when you hover

Reply:
Punting on this for the next polish release